WordPress/Classes/ViewRelated/Views/BottomSheet/BottomSheetPresentationController.swift

```diff
@@ -0,0 +1,251 @@
+
+import UIKit
+
+// MARK: - BottomSheetPresentationControllerConstants
+
+private struct BottomSheetPresentationControllerConstants {
+    static let cornerRadius         = CGFloat(16)
+    static let transitionDuration   = TimeInterval(0.3)
+}
+
+// MARK: - BottomSheetPresentationController
+
+///
+/// BottomSheetPresentationController manages presentation of a view controller like a bottom sheet.
+///
+/// Adapted from AAPLCustomPresentationController, Custom View Controller Presentations and Transitions
+/// https://developer.apple.com/library/archive/samplecode/CustomTransitions/Introduction/Intro.html#//apple_ref/doc/uid/TP40015158
+///
+class BottomSheetPresentationController: UIPresentationController {
```

Comment:
I would suggest defaulting to final, the same way we default to private, and for the same reason: hiding information / behaviour.

Unless you clearly want this class to be subclassed. If that were the case, make it would be good signalling it as open?

Reply:
This is valid feedback. I would like to defer acting on it, however, until we extract this to a framework (e.g., `WordPressUI` or another pod).